test/unit-test/KuksavalUnitTest.cpp

```diff
@@ -2391,56 +2397,6 @@ BOOST_AUTO_TEST_CASE(permission_basic_write_with_branch_permission)
    get_response_json.erase("timestamp");
    BOOST_TEST(get_response_json == get_expected);
 
-   // Now try to read with a valid token.
```

Comment:
why removed?

Reply:
According to the name `permission_basic_write_with_branch_permission` and the idoamtic way this tests are written, this testcase is done here. 
The rest seems unrelated (maybe copy paste residue? It is just a basic read with a basic Permission, but written in slightly different style. And there already is another separate testcase for that behavior